octobot_trading/personal_data/portfolios/assets/future_asset.pxd

```diff
@@ -18,15 +18,17 @@ cimport octobot_trading.personal_data.portfolios.asset as asset_class
 
 
 cdef class FutureAsset(asset_class.Asset):
-    cdef public object equity
+    cdef public object unrealized_pnl
     cdef public object initial_margin
-    cdef public object margin_balance
-    cdef public object maintenance_margin
-    cdef public object position_initial_margin
-    cdef public object unrealised_pnl
+    cdef public object wallet_balance
+    cdef public object position_margin
+    cdef public object order_margin
 
-    cpdef bint update(self, object available=*, object total=*, object initial_margin=*, object margin_balance=*,
-                      object maintenance_margin=*, object position_initial_margin=*, object unrealised_pnl=*)
-    cpdef bint set(self, object available=*, object total=*, object initial_margin=*, object margin_balance=*,
-                   object maintenance_margin=*, object position_initial_margin=*, object unrealised_pnl=*)
-    cdef void _update_equity(self)
+    cpdef bint update(self, object total=*, object available=*, object position_margin=*,
+                      object unrealized_pnl=*, object initial_margin=*)
+    cpdef bint set(self, object total=*, object available=*, object margin_balance=*, object initial_margin=*,
```

Comment:
by returning bint here and in `update`, we will block potential raised exceptions, it's not a problem here ?

Reply:
Yes it's an issue for now. I tried to return object here and in multiple intern method to solve the issue but it didn't work.

Reply:
What was the problem with returning object ? 😮

Reply:
This was not a problem, it was not solving the issue